inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/source/ServerMessageHandler.java

```diff
@@ -442,6 +442,10 @@ private void formatMessagesAndSend(ChannelHandlerContext ctx, Map<String, String
                     headers.put(AttributeConstants.DATA_TIME, String.valueOf(System.currentTimeMillis()));
                 }
 
+                if ("false".equals(commonAttrMap.get(AttributeConstants.MESSAGE_IS_ACK))) {
+                    headers.put(AttributeConstants.MESSAGE_IS_ACK, "false");
+                }
```

Comment:
How about "orderEvent.getHeaders().get(AttributeConstants.MESSAGE_IS_ACK))==null" and other value?

Reply:
just false need to be handle.